include/picongpu/param/species.param

```diff
@@ -88,6 +89,6 @@ using UsedParticleCurrentSolver = currentSolver::Esirkepov< UsedParticleShape >;
  * For development purposes: --------------------------------------------------
  * - particles::pusher::Axel : a pusher developed at HZDR during 2011 (testing)
  */
-using UsedParticlePusher = particles::pusher::Boris;
+using UsedParticlePusher = particles::pusher::HC;
```

Comment:
Please do not change the default for now.

Reply:
Why not? It is supposed to become the new standard as it is the best of Boris' and Vay's method combined. And it is tested and proven to work.

Reply:
We can switch this to the new standard soon. But for now I would prefer having it as a optional pusher so that we can play around with it is simulations using the `dev` branch. @ComputationalRadiationPhysics/picongpu-developers what do you thin? 

This slow approcha allows to observe possible performance penalties and glitches in the behavior of the new pusher before letting everyone else use it by default (without their explicit knowledge of this switch). 

The same will be done with the EmZ algorithm - currently [Esirkepov is still the default ](https://github.com/ComputationalRadiationPhysics/picongpu/blob/fe322db49e69b09258691c4aa781769470fe693d/include/picongpu/param/species.param#L71) despite EmZ being better.

Reply:
I would agree. Maybe have this PR adding a new pusher. And another PR solely to change the default. This way may also be better for release notes, as these two PRs would come as two different lines.